metadata-service/factories/src/main/java/com/linkedin/metadata/boot/kafka/DataHubUpgradeKafkaListener.java

```diff
@@ -85,7 +85,7 @@ public void checkSystemVersion(final ConsumerRecord<String, GenericRecord> consu
       if (expectedVersion.equals(event.getVersion())) {
         IS_UPDATED.getAndSet(true);
       } else {
-        log.info("System version is not up to date: {}. Waiting for datahub-upgrade to complete...", expectedVersion);
+        log.error("System version is not up to date: {}. Waiting for datahub-upgrade to complete...", expectedVersion);
```

Comment:
I think probably warn?

Reply:
I was thinking INFO even, but WARN makes sense so it pops a bit more.